screens/home/HomeScreen.js

```diff
@@ -543,7 +549,7 @@ class HomeScreen extends PureComponent {
                     galleryToUpload.append('date', date);
                     galleryToUpload.append('presence', img.picked_up);
                     galleryToUpload.append('model', model);
-
+                    console.log(JSON.stringify(galleryToUpload, null, 2));
```

Comment:
Log can be removed

Reply:
kept it so that we know from sentry logs if something goes wrong...